module5/preparation/Week4/Swagger.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: page
+title: Swagger API Documentation Prep
+---
+
+As part of this week's Calling an API Mini Project, you will be learning about a new tool called [Swagger](https://swagger.io/).
+
+Watch [this](https://www.youtube.com/watch?v=txArM31my9M) video to get an introduction to Swagger API Documentation.
+
+Feel free do also do some independent research. Swagger is a broad tool that can have a lot of uses, we are focusing on it's ability to create API documentation and an interactive environment for calling an API, as shown in the video.
+
+Be prepared to share answers to the following questions:

```

Comment:
Should they prepare to answer these in this session, or in the lesson?

Reply:
Ope, looks like in the lesson 😆.  Should we maybe add these instructor notes to the lesson also?  That way they are with the discussion?

Reply:
Good call to add this instructor note to the lesson part when we discuss these questions.